src/com/hms_networks/americas/sc/historicaldata/HistoricalDataQueueManager.java

```diff
@@ -1,15 +1,7 @@
 package com.hms_networks.americas.sc.historicaldata;
 
 import com.hms_networks.americas.sc.fileutils.FileAccessManager;
-import java.io.BufferedReader;
```

Comment:
90% of the time you have "and" in a commit title it should be two commits.

Reply:
Ill change that. Also, the google code style file changed again. I configured it to skip checking the import order. It now will only throw an error for unused imports